base_trivy_scan.yml

```diff
@@ -0,0 +1,36 @@
+# https://github.com/aquasecurity/trivy#gitlab-ci
+
+.base_trivy_scan:
+  stage: Deploy
+  image: docker:stable
+  services:
+    - name: docker:dind
+      entrypoint: ["env", "-u", "DOCKER_HOST"]
+      command: ["dockerd-entrypoint.sh"]
+  variables:
+    # See https://github.com/docker-library/docker/pull/166
+    DOCKER_TLS_CERTDIR: ""
+    IMAGE: trivy-ci-test:$CI_COMMIT_SHA
+  before_script:
+    - apk add --no-cache curl
+    - export VERSION=$(curl --silent "https://api.github.com/repos/aquasecurity/trivy/releases/latest" | grep '"tag_name":' | sed -E 's/.*"v([^"]+)".*/\1/')
+    - echo $VERSION
+    - wget https://github.com/aquasecurity/trivy/releases/download/v${VERSION}/trivy_${VERSION}_Linux-64bit.tar.gz
+    - tar zxvf trivy_${VERSION}_Linux-64bit.tar.gz
+  allow_failure: true
+  script:
+    # Build image
+    - docker build -t $IMAGE .
+    # Build report
+    - ./trivy -d --exit-code 0 --cache-dir .trivycache/ --no-progress --format template --template "@contrib/gitlab.tpl" -o gl-container-scanning-report.json $IMAGE
+    # Print report
+    - ./trivy -d --exit-code 0 --cache-dir .trivycache/ --no-progress $IMAGE
+    # Fail on high and critical vulnerabilities
+    - ./trivy -d --exit-code 1 --cache-dir .trivycache/ --no-progress $IMAGE
```

Comment:
We want a socialgouv/docker/trivy image :)

Reply:
theres nothing really specific ?

Reply:
@renovate-bot :eye: https://github.com/SocialGouv/docker/pull/228